services/QuillLMS/engines/evidence/lib/evidence/spelling_check.rb

```diff
@@ -89,7 +91,7 @@ def error
     private def bing_response
       @response ||= HTTParty.get(BING_API_URL.to_s,
         headers: {
-          "Ocp-Apim-Subscription-Key": ENV['OCP-APIM-SUBSCRIPTION-KEY']
+          "Ocp-Apim-Subscription-Key": ENV['BING_SPELL_KEY']
```

Comment:
I changed the env var since env vars with hyphens are silently ignored in our local environments. I've updated staging and prod with this new env var.

Reply:
That's good to know.  Thanks for figuring that out.